### DISCUSSION TITLE
Blank beatmap panels in multiplayer

### DISCUSSION THREAD

**Discussion by Sinamer** [ppy/osu#16136] [Q&A] [answered]
Sometimes beatmap will have a blank panel when added to queue in multiplayer. It is possible to edit it, but the panel still stays blank. Remove button throws an unobserved error. Ready button is not interactive.

https://user-images.githubusercontent.com/53346335/146610027-95685bf3-ab9c-4226-bc5b-c7ce46f4e887.mp4

Logs:
[runtime.log](https://github.com/ppy/osu/files/7737850/runtime.log) (errors start on line 378)
[network.log](https://github.com/ppy/osu/files/7737855/network.log)

**Comment by smoogipoo** [comment]
Strange, there was no beatmap request for that item in the logs...

**Comment by bdach** [comment, marked as answer]
Should be resolved with #16200
